lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -451,9 +454,6 @@
             <?php if ($register_voters) : ?>
               <p><?php print $voter_reg_share_modal_text_2 ?></p>
             <?php endif; ?>
-            <div class="padded-gray-box">
-              <?php print $custom_social_share_link ?>
```

Comment:
yes that makes sense! Should I do a separate PR for that or should I combine here? @ngjo @angaither